### PR TITLE
[RW-2624][Risk=no] Require concepts for valid concept set

### PR DESCRIPTION
**Pull request by blrubenstein**

### CODE REVIEW

File: ui/src/app/views/concept-set-list/component.tsx [hunk 1]
```diff
@@ -48,11 +43,8 @@ export const ConceptSetsList = withCurrentWorkspace()(
     }
 
     componentDidMount() {
-      try {
-        this.getDomainInfo();
-      } finally {
-        this.loadConceptSets();
-      }
+      this.getDomainInfo();
```
Comment (NehaBroad) [inline ui/src/app/views/concept-set-list/component.tsx]: Do we need to log errors if domainInfo returns error?

File: api/src/main/java/org/pmiops/workbench/api/ConceptSetsController.java [hunk 2]
```diff
@@ -253,6 +257,16 @@ private void addConceptsToSet(org.pmiops.workbench.db.model.ConceptSet dbConcept
       String workspaceId, Long conceptSetId, UpdateConceptSetRequest request) {
     org.pmiops.workbench.db.model.ConceptSet dbConceptSet = getDbConceptSet(
         workspaceNamespace, workspaceId, conceptSetId, WorkspaceAccessLevel.WRITER);
+
+    Set<Long> allConceptSetIds = dbConceptSet.getConceptIds();
+    if (request.getAddedIds() != null) {
+      allConceptSetIds.addAll(request.getAddedIds());
+    }
+    int sizeOfAllConceptSetIds = allConceptSetIds.stream().distinct().collect(Collectors.toSet()).size();
```
Comment (NehaBroad) [inline api/src/main/java/org/pmiops/workbench/api/ConceptSetsController.java]: Since allConceptSetIds is  set we do not need distinct()

**Comment by NehaBroad** [comment]
Do we need CreateConceptSetModal anymore? i noticed it being referenced  in dataPage but not sure its being used anymore

File: ui/src/app/views/concept-set-list/component.tsx (hunk 1 above)
Comment (blrubenstein) [reply ui/src/app/views/concept-set-list/component.tsx]: We are, as part of the `getDomainInfo` function.

**Pull request merged**